smpplib/command.py

```diff
@@ -148,7 +148,7 @@ def generate_params(self):
                     value = self._generate_int(field)
                     body += value
                 elif param.type is str:
-                    value = self._generate_string(field)
+                    value = self._generate_string(field).encode()
```

Comment:
Hello!
Thank you for finding this one. We've moved to python3 recently and while some people use the library in production, there still could be some issues.
Please open separate PR for that patch

Reply:
`_generate_string` returns a byte-string already: `return six.b(value)`